nuxt.config.js

```diff
@@ -138,13 +138,26 @@ export default {
   ],
 
   // Modules (https://go.nuxtjs.dev/config-modules)
-  modules: ["@nuxtjs/google-fonts"],
+  // socail-sharing (https://www.npmjs.com/package/vue-social-sharing)
+  modules: ["@nuxtjs/google-fonts", "vue-social-sharing/nuxt"],
 
   // Axios module configuration (https://go.nuxtjs.dev/config-axios)
   axios: {},
 
   // Build Configuration (https://go.nuxtjs.dev/config-build)
-  build: {},
+  build: {
```

Comment:
Let’s not do this in this PR. We need to solve whatever is going on here holistically.

Reply:
assuming you mean the fix that I left commented out? Went ahead and deleted it.